src/modules/ModuleGovernance/ProposalStatus.vue

```diff
@@ -0,0 +1,42 @@
+<script setup lang="ts" name="ModuleGovernanceProposalStatus">
```

Comment:
Maybe `ProposalStatus` should be moved to `Proposal` directory? Looks confusing for me.